app/Services/UsesTelemetry.php

```
@@ -0,0 +1,20 @@
+<?php namespace AgreableTelemetryPlugin\Services;
+
+use TimberPost;
+
+class UsesTelemetry
+{
+    public static function check(TimberPost $post)
+    {
+        $widgets = $post->get_field('widgets');
+        if (empty($widgets)) {
+            return;
+        }
+        foreach ($widgets as $index => $widget) {
+            if ($widget['acf_fc_layout'] === 'telemetry_acquisition') {
+                return $index;
+            }
+        }
+        return false;
```

Also you'd probably throw a custom `Exception` instead of returning null/false. But there's a fair bit of overhead with that. It's probably okay because we don't really do Exception handling often.